src/plugins/intel_cpu/src/nodes/tensoriterator.h

```diff
@@ -85,7 +93,9 @@ class DynamicBuffer {
     size_t count = 1lu;
     size_t elem_size = 0lu;
     ptrdiff_t chunk_offset_in_byte = 0;
-    ptrdiff_t buffer_offset_in_byte = 0;
+    size_t mem_holder_unit = 0;   // the amount of bytes copied per each count per each execution (iteration)
+    size_t num_iters = 0lu;      // number of executions happened
```

Comment:
`execute` method take `iter` as a parameter and it means the number of the tensor iterator iteration. If this number of interactions is different in the sense, so m.b. it does make sense to rename it somehow, so just to avoid confusion.

Reply:
changed.